src/components/results/common/results-global-filter.tsx

```diff
@@ -0,0 +1,270 @@
+                            size={'small'}
+                            label={getOptionLabel(element)}
+                            {...getTagsProps({ index })}
+                            sx={getChipStyle(element)}
+                        />
+                    ))
+                }
+                // renderGroup : the box below that is visible when we focus on the AutoComplete
+                renderGroup={(item) => {
+                    const { group, children } = item;
+                    const itemRecentAvailable = !!group;
+                    return (
+                        <Box
+                            key={'keyBoxGroup_' + group}
+                            sx={mergeSx(
+                                styles.chipBox,
+                                itemRecentAvailable && styles.recentBox
+                            )}
+                        >
+                            {itemRecentAvailable && (
+                                <Box sx={styles.recentLabel}>
+                                    <FormattedMessage id="results.globalFilter.recent" />
+                                </Box>
+                            )}
+                            {children}
+                        </Box>
+                    );
+                }}
+                // renderOption : the chips that are in the box that is visible when we focus on the AutoComplete
+                renderOption={(props, option: Filter) => {
+                    const { children, color, ...otherProps } = props;
+                    return (
+                        <Chip
+                            {...otherProps}
+                            component="li"
+                            label={getOptionLabel(option)}
+                            size="small"
+                            sx={getChipStyle(option)}
+                        />
+                    );
+                }}
+                // Allows to find the corresponding chips without taking into account the recent status
+                isOptionEqualToValue={(option: Filter, value: Filter) =>
+                    option.label === value.label &&
+                    option.filterType === value.filterType
+                }
+                // Allows to find the translated countries (and not their countryCodes) when the user inputs a search value
+                filterOptions={(
+                    options: Filter[],
+                    state: FilterOptionsState<Filter>
+                ) =>
+                    options.filter((option) => {
+                        if (option.filterType === 'country') {
+                            return translate(option.label)
+                                .toLowerCase()
+                                .includes(state.inputValue.toLowerCase());
+                        }
+                        return option.label
+                            .toLowerCase()
+                            .includes(state.inputValue.toLowerCase());
```

Comment:
const labelToMatch = option.filterType === 'country' ? translate(option.lable) : option.label;

return label.ToMatch.toLowerCase().includes(state.inputValue.toLowerCase());

In the future, we have also other enum of properties.. should avoid code duplication..

Reply:
Yes, that's better.